charters/Architecture/charter.md

```diff
@@ -0,0 +1,35 @@
+# DCP Architecture Team
+
+## Description
+
+The Data Coordination Platform (DCP) Architecture Team is responsible for the technical oversight of the DCP, including identifying and addressing *cross-cutting* architectural concerns across DCP projects.
+
+The core team is composed of *Technical Leads* representing each DCP component to ensure a diversity of skills and views needed for community technical decision-making. 
+
+## In-scope
+Strategic responsibilities
+* Set overall **technical** direction for the DCP
+* Publish Architecture Requests for Comments (RFCs) documenting technical debt that impacts delivery of DCP as an integrated system and must be addressed by PM product plans
+* Review and provide final approval for technical charters for DCP projects
+* Review and provide final approval for cross-component RFC(s)
+    * Available to mentor on all RFC designs
+* Develop technical standards and best practices including, but not limited to:
+    * Technology Radar
```

Comment:
what is "technology radar"?   Is this a well-known term (a quick google search didn't help)...

Reply:
https://www.thoughtworks.com/radar/faq companies publish these internally too. We did at SoundCloud.